lib/components/user/notification-prefs-pane.js

```diff
@@ -27,74 +34,78 @@ const Details = styled.div`
 /**
  * User notification preferences pane.
  */
-class NotificationPrefsPane extends Component {
-  static propTypes = {
-    onUserDataChange: PropTypes.func.isRequired,
-    userData: PropTypes.object.isRequired
-  }
-
-  _handleNotificationChannelChange = e => {
-    const { onUserDataChange } = this.props
-    onUserDataChange({ notificationChannel: e })
-  }
+const NotificationPrefsPane = ({
+  errors,
+  handleBlur,
+  handleChange,
+  touched,
```

Comment:
What is the meaning of the name for the `touched` prop?

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/239/commits/fb18f75ff1e1417ab6e8a359b5091fba4cdcc6d3